<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWidget</class>
 <widget class="QWidget" name="GraphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="btn_heading">
       <property name="text">
        <string>Heading</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="btn_lat">
       <property name="text">
        <string>Latitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="btn_lon">
       <property name="text">
        <string>Longtitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="btn_alt">
       <property name="text">
        <string>Attitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="btn_pitch">
       <property name="text">
        <string>Pitch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="btn_roll">
       <property name="text">
        <string>Roll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="btn_yaw">
       <property name="text">
        <string>yaw</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="btn_xvel">
       <property name="text">
        <string>X Velocity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="btn_yvel">
       <property name="text">
        <string>Y Velocity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="btn_zvel">
       <property name="text">
        <string>Z Velocity</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCustomPlot" name="customPlot" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
